components/servo/Cargo.lock

```diff
@@ -183,6 +183,11 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "bitflags"
+version = "0.4.0"
```

Comment:
I'm not sure how to accomplish this. It looks like the ws library pulls in a different version of the bitflag library than servo itself. How would I go about resolving this?